### PR TITLE
Update Google Calendars integration documentation page.

### PR DESCRIPTION
**Pull request by LeoCal**
## Proposed change
Update Google Calendars integration documentation page.

## Type of change
- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
N/A

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/google.markdown
Comment (allenporter) [inline source/_integrations/google.markdown]: The correct name is `Google Calendar` which was fixed in the next release.

Comment (LeoCal) [reply source/_integrations/google.markdown]: Thanks for your input, I have changed Calendars to Calendar.
As what I see on the "current" branch is still wrong, I hope this PR is still good for the Hacktoberfest.

Comment (allenporter) [reply source/_integrations/google.markdown]: Yes, that's fine -- can you just adjust for the merge conflicts?

Comment (LeoCal) [reply source/_integrations/google.markdown]: Sure, I'll do - thanks.

Comment (LeoCal) [reply source/_integrations/google.markdown]: Done, waiting for approval.

**Review by frenck — APPROVED** [review]
Thanks, @LeoCal 👍 

../Frenck

**Pull request merged**
